boards/stm32f429i-disc/doc.txt

```diff
@@ -0,0 +1,36 @@
+/**
+ * @defgroup    boards_stm32f429i-disc STM32F429I-DISCO and -DISC1
+ * @ingroup     boards
+ * @brief       Support for the `STM32F429-DISCO` and `STM32F429-DISC1` boards.
+
```

Comment:
You don't need all those blank lines between sections. One would be enough

Reply:
Yeah, this is just my style of writing markdown I guess. Is there a markdown styleguide for RIOT? I don't recall.

Reply:
> Is there a markdown styleguide for RIOT?

I don't think so. And looking at other files there's no 4 blank lines after the doxygen header in general. The number of blank lines before section is between 1 and 3 in general.
i prefer using the minimum blank lines possible in general but this is personal taste.